lfs/util.go

```diff
@@ -86,3 +88,64 @@ func wrapProgressError(err error, event, filename string) error {
 
 	return nil
 }
+
+// Return whether a given filename passes the include / exclude path filters
+// Only paths that are in includePaths and outside excludePaths are passed
+// If includePaths is empty that filter always passes and the same with excludePaths
+// Both path lists support wildcard matches
+func FilenamePassesIncludeExcludeFilter(filename string, includePaths, excludePaths []string) bool {
```

Comment:
We have no Windows CI yet :/  But yes, very important there. Now that `script/bootstrap` should work on Windows, I'll resume the search for a CI system ([AppVeyor](http://www.appveyor.com/)?).